resources/migrations/000_migrations.yaml

```diff
@@ -15362,6 +15362,50 @@ databaseChangeLog:
                   constraints:
                     nullable: false
 
+  - changeSet:
+      id: v48.00-018
+      author: noahmoss
+      comment: Add new recent_views table
```

Comment:
This is the table to store a limited number of recent views for each user, which was previously in a user-local setting.